app/models/program.py

```diff
@@ -14,6 +14,8 @@ class Program(baseModel):
     contactEmail = CharField(null=True,default='')
     defaultLocation = CharField(null=True,default='')
 
+    OTHERCELTSSPONSORED = 9
```

Comment:
That's not quite a field in the database. That variable is simply meant as a static and constant class variable for the Program class that we can use to refer to the magic number 9. I don't believe we want to make it a program field because that would add it to the database. See line 8 in /app/models/certification.py for an example of how we use it for BONNER inside of Certification.

Reply:
Okay, i do it